servers/dsub/jobs/test/base_test_cases.py

```diff
@@ -306,12 +306,6 @@ def test_query_jobs_by_label(self):
             no_label_job = self.start_job('echo NO_LABEL', name='nolabeljob')
             no_label_job_id = self.get_api_job_id(no_label_job)
 
```

Comment:
Would this change make the local provider flaky? Can't we not query local jobs until they've started? Another idea would be to support multiple statuses on wait_for_job_status

Reply:
I do not believe it will make them flakey. Querying by local jobs should now work even before they've started (after https://github.com/googlegenomics/dsub/issues/82 was resolved). This is separate from the issue from being unable to abort jobs local jobs which are not yet started.